### ISSUE TITLE
NameError at "/opt/splunk/etc/apps/TA-asngen/bin/asngen.py", line 37 : global name 'ProxyHandler' is not defined 

### ISSUE THREAD

**Issue by KaterinaKop** [doksu/TA-asngen#3]
After installing the app and setting proxies an error occured: 
 NameError at "/opt/splunk/etc/apps/TA-asngen/bin/asngen.py", line 37 : global name 'ProxyHandler' is not defined 
I've fixed it by changing importing urllib2 and changing this part:

```
if proxies['http'] is not None or proxies['https'] is not None:
            proxy = urllib2.ProxyHandler(proxies)
            opener = urllib2.build_opener(proxy)
            urllib2.install_opener(opener)

        try:
            url = urllib2.urlopen("https://download.maxmind.com/download/geoip/database/asnum/GeoIPASNum2.zip")
        except:
            raise Exception("Please check app proxy settings")
```
Also I think it's worth stating how to set CIDR-type lookup in splunk, maybe simply by putting this link to splunkanswers: https://answers.splunk.com/answers/5916/using-cidr-in-a-lookup-table.html

Now command works, that's exactly what I needed! Thank you!

**Comment by doksu** [comment]
Thanks very much Katerina! :)

I fixed this a few days ago (cabdea34dbedbe2fc36febc2776912644918fad6) but hadn't yet published to Splunkbase. Thanks very much for reminding me. Hope you don't mind if I credit you on Splunkbase for finding this bug?

The new version (v0.1.2) is now available: https://splunkbase.splunk.com/app/3531/

**Comment by KaterinaKop** [comment]
Sure, you can!:) 
Have a nice weekend)
